packages/terra-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -244,6 +253,9 @@ function WorklistDataGrid(props) {
           onClearSelectedRows();
         }
       } else if (onDisableSelectableRows) {
+        // In order to set focus to the right cell after row selection is turned off, keep
+        // track of cell in grid that had focus when the call was initiated.
+        gridFocusedWhenRowSelectionDisabled.current = true;
```

Comment:
Shouldn't we be able to handle this with the useEffect for the row selection prop?  It doesn't seem like we should need an extra ref variable.

Reply:
You have a focus event.  That would let you know if you received focus.

Reply:
Discussed offline and came up with alternate solution of using the dom.